src/admin-requisition-group-edit/member-facilities.html

```diff
@@ -0,0 +1,57 @@
+<section class="requisition-group-facilities">
+    <form class="form-inline" ng-submit="vm.addFacility()" reload-form>
+        <div class="form-group">
+            <label for="facility-select">{{'adminRequisitionGroupEdit.facility' | message}}</label>
```

Comment:
here, and in all translations in this file, one time binding can be used
{{:: 'adminRequisitionGroupEdit.facility' | message}}